README.md

```diff
@@ -44,7 +49,9 @@ The beginnings of a Sculpin-based style guide creation tool.
 
 *   `npm run deploy`
 
-    This is a task to deploy the static styleguide to GitHub pages. It will deploy to `gh-pages` branch of the repo defined in the `config/paths.js`. Each commit for this process will default to the message: "Updated with Butler - [timestamp]". *Note: When you are deploying, Butler will ask you for your github credentials multiple times. Enter your own github credentials as prompted.*
+    This is a task to deploy the static styleguide to GitHub pages. It will deploy to `gh-pages` branch of the repo defined in the `config/paths.js`. Each commit for this process will default to the message: "Updated with Butler - [timestamp]". 
```

Comment:
Maybe we should add a note that it uses output_prod and may require a sculpin_site_prod.yml file? (or maybe this file should be created by default?)